src/backend/migrations/002-create-geometry-log.sql

```diff
@@ -0,0 +1,19 @@
+-- ## Migration to create a table to store new and bad geometries.
+
+-- Start a transaction
+
+BEGIN;
+
+CREATE TABLE geometrylog (
```

Comment:
Nice work 👍

Don't forget the db init script though =)

Reply:
Oh yeah. okay :+1: